be/src/vec/functions/round.h

```diff
@@ -164,8 +170,8 @@ class DecimalRoundingImpl {
 
 public:
     static NO_INLINE void apply(const Container& in, UInt32 in_scale, Container& out,
```

Comment:
why here we need change the api ?

Reply:
Because for e.g. in line 656 it's called with type `int32_t`:
```
                DecimalRoundingImpl<T, rounding_mode, tie_breaking_mode>::apply(
                        decimal_col->get_element(i).value, input_scale,
                        col_res->get_element(i).value, col_scale_i32.get_data()[i]);
```